### PR TITLE
Design review

### PR DESCRIPTION
**Pull request by ale-romo**
- [ *]  Mostrar todos los días sin scroll horizontal: **Lo dejé con scroll horizontal, se ve bien**
- [x]  Estilizar EventCard
- [x ]  Agregar footer
- [x]  Poner el flaminfo arriba a la derecha. 🦩
- [x]  Ocultar logo chico y activarlo en scroll
- [x]  Eliminar ícono de mapa en la tarjeta.
- [x]  Pasar Nombre (svg) a Texto

### CODE REVIEW

File: app/container.tsx [hunk 1]
```diff
@@ -78,95 +85,120 @@ export default function Home({
...
-            display: "flex",
-            flexShrink: "0",
+            position: 'absolute',
+            top: '1rem',
+            right: '1rem',
+            width: '2rem',
+            cursor: 'pointer',
           }}
         >
+          <Flaminfo />
```
Comment (garciadiazjaime) [inline app/container.tsx:114]: let's remove it, doesn't add much value and it takes some realstate, the landing without it looks very nice; also the main purpose of the text is SEO; we can always revisit later

Screenshot 2024-04-16 at 10 48 19 AM

File: app/container.tsx [hunk 2]
```diff
@@ -78,95 +85,120 @@ export default function Home({
...
-          width: 60vw;
-
-          @media (min-width: ${tokens.breakpoints.md}) {
-            width: 40vw;
-          }
-
-          @media (min-width: ${tokens.breakpoints.lg}) {
-            width: 25vw;
+          h1 {
+            font-size: 2.8rem;
```
Comment (garciadiazjaime) [inline app/container.tsx:173]: can these two be moved to the element? `style={{fontSize, letterSpacing`?

File: app/container.tsx [hunk 3]
```diff
@@ -78,95 +85,120 @@ export default function Home({
...
+              }
+            }
           }
 
           svg:first-child {
             width: 40%;
             margin-left: -15%;
           }
         }
+        .shows {
```
Comment (garciadiazjaime) [inline app/container.tsx:192]: can this be moved to the element?

let's keep it here if it uses: pseudo elements or breakpoints

File: app/layout.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import type { Metadata } from "next";
 
 import StyledJsxRegistry from "./registry";
+import Head from "next/head";
```
Comment (garciadiazjaime) [inline app/layout.tsx:4]: what's this for?

File: components/Nav/index.tsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    const handleScroll = throttle(() => {
+      const scrollY = window.scrollY;
+      const viewportHeight = document.documentElement.clientHeight;
+
+      if (scrollY < viewportHeight - 200) {
+        setIsStuck(false);
+      } else {
+        setIsStuck(true);
+      }
+    }, 100);
```
Comment (garciadiazjaime) [inline components/Nav/index.tsx:18]: why 100?

File: package.json [hunk 6]
```diff
@@ -26,7 +26,9 @@
     "commander": "^11.1.0",
     "csvtojson": "^2.0.10",
     "dotenv": "^16.3.1",
+    "global": "^4.4.0",
```
Comment (garciadiazjaime) [inline package.json:29]: what's this?

File: yarn.lock
Comment (garciadiazjaime) [inline yarn.lock:1]: we don't use `yarn`, let's remove this file and try to use `npm i` whenever installing packages

File: components/Nav/index.tsx (hunk 5 above)
Comment (ale-romo) [reply components/Nav/index.tsx:18]: It's the slowest it could go without feeling slow

**Pull request merged**